fil-proofs-tooling/Cargo.toml

```diff
@@ -25,7 +25,7 @@ chrono = { version = "0.4.7", features = ["serde"] }
 memmap = "0.7.0"
 bellperson = "0.9.1"
 paired = "0.20.0"
-fil-sapling-crypto = "0.6.0"
+fil-sapling-crypto = "0.6.3"
```

Comment:
Rust oddity, these individual patch version updates can be omitted.  They don't hurt though ;-)